src/main/webapp/signup.html

```diff
@@ -18,7 +18,11 @@
     
     <title>Sign Up | Mintern</title>
   </head>
+<<<<<<< HEAD
+  <body onload="loadSignup()">
+=======
   <body onload="isUserRegistered()">
+>>>>>>> f253fb6b66d9b5627f249ac777e981a7faef651a
```

Comment:
you've got to delete these lines from a merge

Reply:
Done!